src/import_map.json

```diff
@@ -0,0 +1,7 @@
+{
+  "imports": {
+    "/": "./",
+    "./": "./",
```

Comment:
d: why is this required? 
Also is `std/fmt/` even used? (I closed some files already but can not see it in the still open files, so I am wondering if this has any effect at all).

Reply:
I stole that trick from the deno manual, it allows absolute import paths https://deno.land/manual/linking_to_external_code/import_maps

I really like those over relative import paths, they make moving code around much easier (e.g. if I move a file from `foo/bar` up one level to `foo/`, I don't have to fix all my relative import paths from `../../` to `../`